test/build/50_ch-image.bats

```diff
@@ -63,6 +63,30 @@ EOF
     [[ $output = *"00_tiny"* ]]
 }
 
+@test 'ch-image reset' {
+   export CH_IMAGE_STORAGE="$CH_IMAGE_STORAGE"/reset
```

Comment:
I'd use a path in the Bats temporary directory (there is a variable for this). Otherwise great test!